rebar.config

```diff
@@ -1,9 +1,33 @@
-{erl_opts, [debug_info, warn_untyped_record]}.
+%% -*- erlang -*-
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2011-2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
 {erl_first_files, ["src/lager_util.erl"]}.
+
+{erl_opts, [
+    debug_info,
+    warn_untyped_record
+]}.
 {deps, [
-        {goldrush, "0\.1\.6",
-            {git, "git://github.com/DeadZen/goldrush.git", {tag, "0.1.6"}}}
-       ]}.
+    {goldrush, ".*", {git, "git://github.com/DeadZen/goldrush.git", {branch, "master"}}}
```

Comment:
One of the first things I did in all repos in which I created OTP-17 branches was to use develop/master (as appropriate) for their dependencies, since lots of them included relevant OTP-17 patches. I didn't evaluate each separately unless I ran into an issue, and took the approach that we might subsequently rein them in if we needed to, but I wanted to get the current work in them.

Riak EE uses locked-deps for releases, so unless there's a reason *not* to use the current work,  I'd rather pick up patches as the maintainers apply them. That said, I don't assume they'll all stay "unpinned" - this is a development branch, after all.